src/grid/Col/index.jsx

```diff
@@ -4,10 +4,11 @@ import React, { createElement } from 'react';
 import PropTypes from 'prop-types';
 import throttle from 'lodash/throttle';
 import getStyle from './style';
-import { getConfiguration } from '../../config';
+import { getConfiguration, ScreenContextConsumer } from '../../config';
 import { getScreenClass } from '../../utils';
 import { NoGutterContext } from '../Row';
 
+@ScreenContextConsumer
```

Comment:
Can't we have a `ScreenClassConsumer`? In the end, only the `rect.width` is actually used, so by providing a screen class directly, we don't have to call `getScreenClass` all the time.